src/libs/address.js

```diff
@@ -1,30 +1,41 @@
 import IdunnPoi from '../adapters/poi/idunn_poi';
 
 /**
- * Format an address given an address object (name, city, country, label, and other regions)
+ * Filter an address and return an array with the relevant items
  * @param {*} address - an address object
  */
-export function format(address) {
-  if (!address) {return '';}
+export function filter(address) {
```

Comment:
I'm not sure about the name, what about something like "extractAddressFields" or 'extractAddressComponents` ?

Reply:
Do not forget to add a new unit test :) It should be straightforward

Reply:
I agree about the name. `toArray` or maybe `formatMultiline` ?

Reply:
ok for toArray, as this function is not only used for multiline but also in format(address, ",") !